src/utils/client.coffee

```diff
@@ -0,0 +1,59 @@
+getToken = (callback)->
+    receiveToken = (event) ->
+        window.removeEventListener 'message', receiveToken
+        {appName, token}  = event.data
+        callback? null, {appName, token}
+        callback = null
+    
+    unless window.parent?
+        return callback new Error('no parent window')
+    
+    unless window.parent?.postMessage
+        return callback new Error('get a real browser')
+    
+    window.addEventListener 'message', receiveToken, false
+    window.parent.postMessage { action: 'getToken' }, '*'
+
+module.exports =
+    get: (path, attributes, callback)->
+        playRequest 'GET', path, attributes, (error, body, response) ->
+            callback error, body, response
```

Comment:
I think we should be coherent with request-json(-light) and put the response before the body.